From 56bf0d707ad8d920798532f507766908d504df8f Mon Sep 17 00:00:00 2001
From: Alessandro Astone <alessandro.astone@canonical.com>
Date: Wed, 18 Jun 2025 12:04:10 +0200
Subject: [PATCH] Drop udev rules

The sole purpose of this set of udev rules was to conditionally disable
Wayland. With commit 38fa8b947f8e8a34a295d581ffa975239be5d1ef dropping the
`gdm-runtime-config` call, there is no logic left here.

Part-of: <https://gitlab.gnome.org/GNOME/gdm/-/merge_requests/299>
---
 data/61-gdm.rules.in | 56 --------------------------------------------
 data/meson.build     | 11 ---------
 2 files changed, 67 deletions(-)
 delete mode 100644 data/61-gdm.rules.in

diff --git a/data/meson.build b/data/meson.build
index 6f0ff7861..5d0703ef7 100644
--- a/data/meson.build
+++ b/data/meson.build
@@ -140,17 +140,6 @@ foreach _pam_filename : pam_data_files
   )
 endforeach
 
-if have_x11_support
-  configure_file(
-    input: '61-gdm.rules.in',
-    output: '@BASENAME@',
-    configuration: {
-      'libexecdir': gdm_prefix / get_option('libexecdir'),
-    },
-    install_dir: udev_dir,
-  )
-endif
-
 # DBus service files
 service_config = configuration_data()
 service_config.set('sbindir', gdm_prefix / get_option('sbindir'))
-- 
GitLab

